build.gradle

```diff
@@ -55,24 +55,21 @@ defaultTasks 'distZip'
 
 repositories {
     mavenCentral()
-    mavenLocal()
-    maven {
-        url "https://maven.atlassian.com/3rdparty/"
-    }
 }
 
 dependencies {
-    implementation 'log4j:log4j:1.2.17',
-            'org.apache.httpcomponents:httpclient:4.2.6',
+    implementation 'org.apache.httpcomponents:httpclient:4.2.6',
             'org.slf4j:slf4j-api:1.7.5',
-            'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.12.2'
+            'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.12.2',
+            'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.12.2'
```

Comment:
I don't see the actual jersey-client library here (`com.sun.jersey:jersey-client:1.19.4`), and we definitely use that - it gets pulled in as a transient, but we should declare it

Reply:
done